src/main/java/com/google/sps/data/BusinessesService.java

```diff
@@ -33,10 +41,43 @@
   /** Create a new Businesses instance
   * @param businesses businesses from SmallCityService
   **/
+
   public BusinessesService(List<Listing> businesses) {
     this.businesses = businesses;
   }
 
+  public PreparedQuery connectToBigBusinessDatabase(){
```

Comment:
nit: Perhaps "getBigBusinesses" may provide more context?

Reply:
fixed this naming convention